docs/source/development.md

```diff
@@ -40,3 +40,25 @@ pixi shell --environment dev
 The environment `dev` contains the most critical development dependencies. However, you can view the available environment sets in the `pixi.toml` to start a different environment if you would like. For example, if you are only updating the docs, you can do this by setting the environment flag to `docs`.
 
 While in the `pixi` shell and/or Docker container, all changes made in the repository or to the `~/.bashrc` file persist in your local file system. You can also use `git` with no issue to push changes.
+
+## Building and Running C++ Code
+We can easily run C++ code using `Pixi`.
+From within the `dev` enviroment, the available commands are:
```

Comment:
Either combine these into one line or add whitespace between them to make it a separate paragraph.